config/task.go

```diff
@@ -226,6 +227,20 @@ func (c *TaskConfig) Finalize(bp *BufferPeriodConfig, wd string) {
 		c.TFVersion = String("")
 	}
 
+	bp := globalBp
+	if _, ok := c.Condition.(*ScheduleConditionConfig); ok {
+		// disable buffer_period for schedule condition
+		if c.BufferPeriod != nil {
+			log.Printf("[WARN] (config.task) disabling buffer_period for "+
```

Comment:
will update this log with new logging format after https://github.com/hashicorp/consul-terraform-sync/issues/377 is merged

Reply:
Updated log line to:

```
logging.Global().Named(logSystemName).Named(taskSubsystemName).Warn(
	"disabling buffer_period for schedule condition. overriding "+
	"buffer_period configured for this task",
	"task_name", StringVal(c.Name), "buffer_period", c.BufferPeriod.GoString())
```
Output:
2021-09-08T12:50:12.631-0400 [WARN]  config.task: disabling buffer_period for schedule condition. overriding buffer_period configured for this task: task_name=periodic10_task buffer_period="&BufferPeriodConfig{Enabled:true, Min:8s, Max:0s}"